### PR TITLE
Cascader: 增加内容超出时前端显示省略号功能(#15684)

### PR DESCRIPTION
**Pull request by Aaminly**
Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md) | [Español](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

File: examples/docs/zh-CN/cascader.md [hunk 1]
```diff
@@ -669,6 +669,252 @@
   };
 </script>
 ```
+
+:::
+
+### 在内容超出前端显示省略号
```
Comment (island205) [inline examples/docs/zh-CN/cascader.md]: 其他语言的文档也需要更新，法语和西班牙语复制英文文档过去。

File: examples/docs/zh-CN/cascader.md [hunk 2]
```diff
@@ -1401,6 +1647,7 @@
 | change-on-select | 是否允许选择任意一级的选项 | boolean | — | false |
 | size | 尺寸 | string | medium / small / mini | — |
 | before-filter | 筛选之前的钩子，参数为输入的值，若返回 false 或者返回 Promise 且被 reject，则停止筛选 | function(value) | — | — |
+| before-ellipsis | 是否在选中项内容超出时前面显示省略号 | boolean | — | false |
```
Comment (island205) [inline examples/docs/zh-CN/cascader.md]: `before-ellipsis` 有点奇怪

**Comment by WaSheep** [comment]
1. please fix the conflict
2. add the unit test
3. add the internationalization doc

**Comment by island205** [comment]
I'll close this PR since it's been a while since it's been opened. Feel free to reopen if you have updates on this

**Pull request closed without merging**